Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ruby:2.4.1
+ENV HOME=/usr/src/quantum
```

Comment:
Is there a way to set this as an environment variable, so that we don't have to require everyone to have the code in `/usr/src/quantum`?